decidim-elections/lib/decidim/votings/polling_officer_zone_engine.rb

```diff
@@ -31,6 +36,12 @@ def load_seed
                         if: Decidim::Votings::PollingOfficer.polling_officer?(current_user)
         end
       end
+
+      initializer "decidim_votings_polling_officer_zone.assets" do |app|
+        app.config.assets.precompile += %w(
+          decidim_votings_polling_officer_zone_manifest.js
+        )
+      end
```

Comment:
This also shouldn't be necessary anymore with webpack, although I see that all the other engines still have it (even though the files they refer to do not exist). We can investigate this and remove it in a separate PR if needed :)